_common.scss

```diff
@@ -56,4 +55,12 @@
         padding-bottom: 0.5rem;
         font-size: 0.875rem;
     }
+    th.mat-header-cell, td.mat-cell, td.mat-footer-cell, 
+    th.mat-header-cell:first-of-type, td.mat-cell:first-of-type, td.mat-footer-cell:first-of-type, 
+    th.mat-header-cell:last-of-type, td.mat-cell:last-of-type, td.mat-footer-cell:last-of-type, 
+    [dir=rtl] th.mat-header-cell:first-of-type:not(:only-of-type), [dir=rtl] td.mat-cell:first-of-type:not(:only-of-type), [dir=rtl] td.mat-footer-cell:first-of-type:not(:only-of-type),
+    [dir=rtl] th.mat-header-cell:last-of-type:not(:only-of-type), [dir=rtl] td.mat-cell:last-of-type:not(:only-of-type), [dir=rtl] td.mat-footer-cell:last-of-type:not(:only-of-type) {
```

Comment:
What's with all of these rules? Do we really need to specify these all individually? Is a less-specific css rule not enough to override the angular material defaults?

Reply:
Without an `important` keyword, it would be pretty hard to do so. This is why we had all the back and forth conversation on the top.

Reply:
These styles are pretty ugly but if this is what we have to do to avoid adding the !important keyword, I'm fine with it.  As long as we avoid it, our adopters can use !important as needed and it'll consistently work.